resolwe_bio/processes/import_data/basespace.yml

```diff
@@ -0,0 +1,50 @@
+- slug: basespace-import
+  name: BaseSpace Files Import
+  requirements:
+    expression-engine: jinja
+    executor:
+      docker:
+        image: resolwebio/utils
+    resources:
+      network: true
+      secrets: true
+  data_name: 'BaseSpace Import ({{file_ids|length}} {% if file_ids|length == 1 %}file{% else %}files{% endif %})'
```

Comment:
Use spaces between curly brackets and the content inside. See an example from the [documentation](http://jinja.pocoo.org/docs/2.10/templates/#variables). Fix this in other places as well.